test/dtypes/test_affine_quantized.py

```diff
@@ -70,5 +75,21 @@ def test_to_device(self):
 
 
 
+class TestAffineQuantizedFloat(TestCase):
+    @unittest.skipIf(not torch.cuda.is_available(), "Need CUDA available")
+    def test_weights_only(self):
+        for apply_quant in [float8_weight_only()]:
```

Comment:
you can add `float8_weight_only()` to this list directly I think: https://github.com/pytorch/ao/blob/eb47c93ca2ae41f85a0d5d73441ad1bec7cb2405/test/dtypes/test_affine_quantized.py#L58